modules/datastore/src/PostImportResult.php

```diff
@@ -51,37 +50,86 @@ class PostImportResult {
    */
   protected ResourceMapper $resourceMapper;
 
-  /**
-   * The PostImport service.
-   */
-  protected PostImport $postImport;
-
   /**
    * The PostImportResult.
    */
   protected array $postImportResult;
 
   /**
-   * PostImportResource constructor.
+   * PostImportResult constructor.
    */
   public function __construct(
     $postImportResult,
```

Comment:
Let's try to pass as arguments a DataResource object and two strings, rather than an arbitrary associative array.